plugins/kubernetes-backend/src/service/KubernetesFetcher.ts

```diff
@@ -224,13 +224,14 @@ export class KubernetesClientBasedFetcher implements KubernetesFetcher {
       [url, requestInit] = this.fetchArgsInCluster(credential);
     } else if (
       credential.type === 'bearer token' ||
+      credential.type === 'x509 client certificate' ||
```

Comment:
This if/elseif/else is becoming a bit of a smell, could we refactor this to make it more readable?

Reply:
sure !

Reply:
Hello Matthew, I have modified the code. it's more readable but it is still a if/elseif/else block, let me know if you consider a better approach

Reply:
Looks better thanks!